core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -55,7 +55,7 @@ public SetupWizard(Jenkins j) throws IOException {
             
             HudsonPrivateSecurityRealm securityRealm = new HudsonPrivateSecurityRealm(false, false, null);
             j.setSecurityRealm(securityRealm);
-            String randomUUID = UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH);
+            String randomUUID = System.getProperty("jenkins.install.initialAdminPassword", UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH));
```

Comment:
@daniel-beck hope to make it in jenkins, if you ever run the installer, you can add a command line parameter to set the password instead of copy/pasting a random one every time